ajax/libs/prop-types/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "prop-types",
+  "version": "15.5.10",
+  "filename": "prop-types.min.js",
+  "description": "Runtime type checking for React props and similar objects.",
+  "homepage": "https://facebook.github.io/react/",
```

Comment:
I am wondering if https://facebook.github.io/react/docs/typechecking-with-proptypes.html is better choice. And there is a question: React.PropTypes is deprecated as of React v15.5 but now react is v15.6.1. If the team releases the newer version in the future. The homepage might be changed.